cryptographic_estimators/MQEstimator/MQAlgorithms/dinur2.py

```diff
@@ -60,41 +55,37 @@ def __init__(self, problem: MQProblem, **kwargs):
     @optimal_parameter
     def n1(self):
         """
-        Return the optimal parameter $n_1$
+        Returns the optimal parameter `n_1`.
```

Comment:
Here we highlight the parameter with backticks. Is there a reason for doing this, because the other parameters dont have it? Does it look better in the documentation?

Reply:
I'm still not sure about the impact, so I decided to stick with what was in the previous docstring in most cases. I can get back to this once we finish the transition!